gnmvidispine/vidispine_api.py

```diff
@@ -259,8 +259,8 @@ def sendAuthorized(self,method,url,body,headers,rawData=False):
         """
         import time
         attempt = 0
-        str = '%s:%s' % (self.user, self.passwd)
-        auth = base64.encodebytes(str.encode("UTF-8")).decode().replace('\n', '')
+        str_for_user = '%s:%s' % (self.user, self.passwd)
+        auth = base64.encodebytes(str_for_user.encode("UTF-8")).decode().replace('\n', '')
```

Comment:
not sure we should actually need the .encode().decode() chain here but am willing to let it pass